faq/dedibox-in-scaleway-console.mdx

```diff
@@ -24,99 +25,103 @@ Once the process is complete, you will be able to manage your Dedibox Servers an
 
-<Message type="important">
+### Can I still use the Dedibox console after the upgrade?
 
-  Once you have linked your account, you can edit your payment information from the Scaleway console only. You will not be able to edit the payment information from the Dedibox console anymore.
+Yes, you can continue to use the Dedibox console to manage all other Online.net services, as well as your Dedibox servers.
 
-</Message>
+### When exactly will the Dedibox console (console.online.net) stop being available?
 
-## Will I have a single login in the future?
+The current Dedibox console will remain accessible.
 
-In the future, you may only need to use your Scaleway login information to connect to the Scaleway console and to manage all your Scaleway products.
+## Billing and payment information
 
-## What impact will linking accounts have on my billing?
+### What impact will linking accounts have on my billing?
 
-You will continue to receive an invoice for each type of product (one for Scaleway Dedibox and one for Scaleway Cloud products).
-However, it is the payment method defined in your Scaleway console that allows you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.<br />
+You will continue to receive an invoice for each type of product (one for Scaleway Dedibox and one for Scaleway Cloud products). However, it is the payment method defined in your Scaleway console that allows you to pay all your Scaleway invoices (Scaleway Dedibox and Scaleway Cloud). It is thus important that following the linking process, you verify whether the default payment method defined in the Scaleway console is your preferred payment method for future invoices.
 
-## Where can I find my Dedibox invoices, following the linking of my accounts?
+### Where can I find my Dedibox invoices, following the linking of my accounts?
 
 You will be able to find all your invoices in the Billing section of the Scaleway console (old Dedibox invoices as well as all new invoices).
 
-## Following the linking of my accounts, will I find my Dedibox payment methods in the Scaleway console?
+### Following the linking of my accounts, will I find my Dedibox payment methods in the Scaleway console?
+
+All your active Dedibox payment methods will be migrated to the new console during the linking process so that you will be able to manage them from the Scaleway console instead of the Dedibox console. You have to choose a unique default payment method for your Scaleway services. This default payment method allows you to pay all of your Scaleway invoices (Scaleway Cloud and Scaleway Dedibox).
```

Comment:
```suggestion
All your active Dedibox payment methods will be migrated to the new console during the linking process so that you will be able to manage them from the Scaleway console instead of the Dedibox console. You must choose a unique default payment method for your Scaleway services, which will be used to pay all your invoices (Scaleway Cloud and Scaleway Dedibox).
```